nvhpc/Dockerfile

```
@@ -0,0 +1,14 @@
+ARG REPOSITORY
+FROM ${REPOSITORY}:base
+LABEL maintainer="Felix Thaler <thaler@cscs.ch>"
+
+ARG NVHPC_VERSION
+ENV NVHPC_VERSION ${NVHPC_VERSION}
+
+RUN NVHPC_VERSION_DASH=$(echo ${NVHPC_VERSION} | sed 's/\./-/g') && \
+    echo 'deb [trusted=yes] https://developer.download.nvidia.com/hpc-sdk/ubuntu/amd64 /' | tee /etc/apt/sources.list.d/nvhpc.list && \
+    apt-get update -y && \
+    apt-get install -y nvhpc-${NVHPC_VERSION_DASH}
```

Some details to be closer to other Dockerfiles; add `DEBIAN_FRONTEND=noninteractive` and delete `/var/lib/apt/lists/*`.
```suggestion
    apt-get update -qq && \
    DEBIAN_FRONTEND=noninteractive apt-get install -qq -y nvhpc-${NVHPC_VERSION_DASH} && \
    rm -rf /var/lib/apt/lists/*
```